_includes/navigation.html

```diff
@@ -1,40 +1,19 @@
-<nav class="sidebar-navigation">
-  <section class="sidebar-navigation--logo">
-    <a href="/"><img src="/assets/icons/turing_logo_with_name.png"
-         alt="Turing School of Software and Design"
-         class="turing-logo"></a>
-  </section>
-  <div class="sidebar-navigation--links">
-    <section class="sidebar-navigation--search">
+<nav role="navigation">
+  <ul class="nav-links">
+    <li>
+      <a href="/" class="home-logo-link">
+        <img src="/assets/icons/TuringSchool_LogoMark_Gray.png"
+         alt="Turing School of Software and Design logo">
+      </a>
+    </li>
+    <li><a href="/today">CALENDAR</a></li>
+    <li><a href="/lessons">LESSONS</a></li>
+    <li><a href="/projects">PROJECTS</a></li>
```

Comment:
Would it be too difficult to add a CSS property for doing a text-transform for uppercase, rather than typing these in CAPS LOCK? This is the best practice that I've seen in the past.